Utilities/UtilConstants.cs

```diff
@@ -6,6 +6,17 @@
 
 namespace Scarlet.Utilities
 {
+    /// <summary> Class to store constants </summary>
+    public class Constants
+    {
+        // PHYSICAL CONSTANTS
+        public const float GRAVITY = 9.80655f; // meters per second per second
+        public const float EARTH_RADIUS = 6371000; // meters
```

Comment:
These aren't used, are they? Kinda random.

Reply:
They are used for some other software teams. E.G. GPS coordinate route mapping I believe. I wanted to put in that other constant, but didn't want to make a class for just that. Also if we implement an accelerometer for example, having that gravitational constant would be useful if we wanted to do a conversion before it goes to the user.

Reply:
Ah, OK. That makes sense. Document 'em! :P